src/map/status.cpp

```diff
@@ -10968,7 +10968,7 @@ static bool status_change_start_post_delay(block_list* src, block_list* bl, sc_t
 #ifdef RENEWAL
 			val4 = val1 * 2; // Chance of casting
 #else
-			val4 = 5 + val1*2; // Chance of casting
+			val4 = 5 + val1*2; // Chance of casting for non-players
```

Comment:
Does this even work? It looks to me like you always pass 0 as val1 in pre-re? How do you set it for non-players?

Reply:
It's set at cast end for non sd
https://github.com/rathena/rathena/blob/66a4a2ab68ce85d781514c3c000433f981d6842f/src/map/skill.cpp#L10250